src/lsp/sql_ast/sql_ast.ml

```diff
@@ -0,0 +1,241 @@
+(*To read for help https://www.microfocus.com/documentation/visual-cobol/30pu9/VC_30PU9_for_Eclipse_for_Windows_WH/HRDBRHESQL25.html*)
+type sql_var_id = string 
+and cobol_var_id = string
+
+and variable = 
+  | SqlVar of sql_var_id
+  | CobolVar of cobol_var_id
+
+and esql_instuction =
+  | At of variable * esql_instuction 
```

Comment:
Usually we add the source location to variables : replace the type `variable` by `variable with_loc`.
Then, you need to update your code to use `Srcloc.payload` (or the operator `~&`) when you need to access the variable.
Note that you need to open `Cobol_common.Srcloc.INFIX` to use the operator.